docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -206,35 +206,53 @@ V&V Checks:
 
 
 .. list-table:: Intrapartum Decision Tree
-  :widths: 3 15 15
+  :widths:  3 5 10 10 15
   :header-rows: 1
 
   * - ID
     - Decision Information 
+    - Data Value 
+    - Source
     - Notes
   * - 0
-    - XX% of simulants have ectopic pregnancies, abortion or miscarriage
-    - These simulants will NOT continue in the model
+    - Incidence of ectopic pregnancies, abortion or miscarriage
+    - get_draws(gbd_round_id=7, location_id=location_id, gbd_id_type='cause_id', gbd_id=[995,374], source='como', measure_id=6, metric_id=3, age_group_id=24, sex_id=2, year_id=2021, decomp_step='iterative')
+    - incidence_c374 + incidence_c995
+    - These simulants will NOT continue in the model. Use the `total population incidence <Total Population Incidence Rate>`_ rate directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence.
   * - 1
-    - XX% of simulants to attend each delivery facility type, based on their propensity 
-    - Several details are still outstanding including: types of delivery facilities modeled, will facility propensity vary with age, subnational location or upstream factors, will ANC care propensity be correlated with delivery facility propensity
+    - % of simulants to attend each delivery facility type, based on their propensity
+    - At home (68.3%), in public/governmental health facility (26.6%), in private and/or NGO health facility (3.3%), and other (4.8%) 
+    - DHS for each location; placeholder values are from `this Ethiopia paper <https://link.springer.com/article/10.1186/s12884-020-03002-x#Tab2>`_.
+    - Denominator in DHS is all births (live and stillbirths) to interviewed women in the 2 years preceding the survey. The above values are placeholders until we do a more in-depth analysis. We would like this to be location specific, please code accordingly. 
   * - 2
     - Need to figure out how we will determine which simulants need a c-section
+    -
     - 
+    -    
   * - 3
-    - XX% of each facility type have cesarian section capabilities
-    -  
+    - % of each facility type have cesarian section capabilities
+    - Public/governmental health facility (62%), private and/or NGO health facility (73%), at home  (0%), other (50%) 
+    - EmONC (Ethiopia; saved at J:\Project\simulation_science\optimization_model_data\EmONC 2016_Master Dataset_Final\ETH_EmONC Assessment 2016_Final Report Jan11 2018.pdf) 
```

Comment:
It looks like backslashes aren't rendering because they're a special character, so you'll have to [escape them using a double backslash](https://stackoverflow.com/questions/57265111/escape-special-characters-in-restructuredtext-sphinx#comment131155294_57266065) `\\`.

Also, this single long filepath string with no spaces is making this table column very wide... Not sure what the best solution is -- would it look ok to put a space after each backslash?

Reply:
Ah, I see. I also could delete the file path from here, as it's saved in Sharepoint also and I don't think it really needs to be here!

Reply:
Seems reasonable -- you could add a link to the Sharepoint file instead.